x-pack/legacy/plugins/infra/public/pages/logs/analysis/setup/initial_configuration_step/analysis_setup_timerange_form.tsx

```diff
@@ -105,27 +111,44 @@ export const AnalysisSetupTimerangeForm: React.FunctionComponent<{
           <EuiFormControlLayout
             clear={endTime ? { onClick: () => setEndTime(undefined) } : undefined}
           >
-            <EuiDatePicker
-              showTimeSelect
-              selected={endTimeValue}
-              onChange={date => setEndTime(selectedDateToParam(date))}
-              placeholder={endTimeDefaultDescription}
-              openToDate={now}
-              minDate={startTimeValue}
-              minTime={
-                selectedEndTimeIsToday
-                  ? now
-                  : moment()
-                      .hour(0)
-                      .minutes(0)
-              }
-              maxTime={moment()
-                .hour(23)
-                .minutes(59)}
-            />
+            <WithFixedDatepickerZIndex>
+              {popperClassName => (
+                <EuiDatePicker
+                  showTimeSelect
+                  selected={endTimeValue}
+                  onChange={date => setEndTime(selectedDateToParam(date))}
+                  placeholder={endTimeDefaultDescription}
+                  openToDate={now}
+                  minDate={startTimeValue}
+                  minTime={
+                    selectedEndTimeIsToday
+                      ? now
+                      : moment()
+                          .hour(0)
+                          .minutes(0)
+                  }
+                  maxTime={moment()
+                    .hour(23)
+                    .minutes(59)}
+                  popperClassName={popperClassName}
+                />
+              )}
+            </WithFixedDatepickerZIndex>
           </EuiFormControlLayout>
         </EuiFlexGroup>
       </EuiFormRow>
     </EuiDescribedFormGroup>
   );
 };
+
+const WithFixedDatepickerZIndex = euiStyled(
```

Comment:
I created this function-as-a-child component to work around the issue that the popover class must be specified as `popperClassName` instead of `className`.